reference/04-Patterns/synced-pattern-overrides.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 3
+---
+
+# Synced Pattern Overrides
+
+Editors now have the capability to maintain a Synced Pattern while being able to edit the content of the block on an individual basis.  In other words, we can keep the look and feel of our component in sync across the site, while being able to alter the text, links, or even images used within on a per use basis.
+
+It should be noted that only a handful of core blocks allow for this, and custom block support does not currently exist. As of WordPress 6.6, the following blocks are enabled with the supported attributes listed after each one.
+
+| Supported Blocks | Supported Attributes |
+| ---------------- | ---------------------|
+| Image            | url, alt, title      |
+| Paragraph        | content              |
+| Heading	       | content              |
+| Button           | url, text, linkTarget, rel |
+
+In the case of our Call to Action pattern, we can override the Heading, Paragraph, and Button used *within*, but not overide the settings as a whole on the parent Group for example.
+
+:::caution
+When using Synced Pattern Overrides, it is very important to know that our blocks are not optional, and we cannot display them conditionally.
+
+For example, we cannot use multiple Buttons in our pattern and have the ability to sometimes display just a single one.  Every block used in the original pattern will *always* render its markup, even if we leave its text content empty.
```

Comment:
I would like to add how it's not possible to add additional list items to a list. I think that best showcases the shortcomings here. 
﻿
And again if we can link to relevant issues on GitHub that would be even better :)

Reply:
Done!